bot/seasons/evergreen/nationday.py

```diff
@@ -0,0 +1,162 @@
+import datetime
+import json
+import logging
+from pathlib import Path
+from typing import List, Tuple
+
+import discord
+from aiohttp import ClientSession
+from discord.ext.commands import Bot, Cog, Context, command
+
+from bot.pagination import ImagePaginator
+
+
+logger = logging.getLogger(__name__)
+# RESTful API to get countries info
+url = "https://restcountries.eu/rest/v2/alpha/"
+# white flag image to show during error
+white_flag = (
+    "https://upload.wikimedia.org/wikipedia/commons/thumb/a/a7/"
+    "White_flag_waving.svg/158px-White_flag_waving.svg.png"
+)
+# url to get images of flags
+flag_url = "https://www.countryflags.io/{country_code}/flat/64.png"
+
+
+class NationDay(Cog):
+    """NationDay Cog that to help users get info on countries powered by the RESTCountries API."""
+
+    def __init__(self, bot: Bot):
+        self.bot = Bot
+        self.http_session: ClientSession = ClientSession()
+        with open(Path("bot/resources/evergreen/nationday/countries_by_day.json"), "r") as file:
+            self.by_day = json.load(file)
+
+        with open(Path("bot/resources/evergreen/nationday/iso_codes.json"), "r") as file:
+            self.iso_codes = json.load(file)
+
+    async def get_key(self, val: str) -> str:
+        """Get date for particular country, if present."""
+        for key, value in self.by_day.items():
+            if val in value:
+                return key
+        return None
+
+    async def get_specific_country(self, country: str) -> Tuple[List[Tuple[str, str]], str]:
+        """
+        Get Indepedence Day of given country.
+
+        Return Indepedence Day and page of information on country.
+        """
+        # Get date of specified country from dict
+        date = await self.get_key(country)
+        if date:
+            page, img = await self.get_country_info(self.http_session, country)
+            return [(page, img)], date
+        return [(None, None)], None
```

Comment:
I just made it that way so that it's consistent with the return type.
Should just change it to `return None, None` ? Cause when it's called, I unpack it into two variables.